cadctl/cmd/investigate/investigate.go

```diff
@@ -231,3 +237,16 @@ func updateMetrics(investigationName string, result *investigation.Investigation
 		metrics.Inc(metrics.LimitedSupportSet, append([]string{investigationName}, result.LimitedSupportSet.Labels...)...)
 	}
 }
+
+func updateIncidentTitle(pdClient *pagerduty.SdkClient) error {
+	currentTitle := pdClient.GetTitle()
+	if strings.Contains(currentTitle, pagerdutyTitlePrefix) {
+		return nil
+	}
+	newTitle := fmt.Sprintf("[CAD Investigated] %s", pdClient.GetTitle())
```

Comment:
Let's just use the existing `currentTitle` value here to save an API call
```suggestion
	newTitle := fmt.Sprintf("[CAD Investigated] %s", currentTitle)
```

Reply:
Whoops, good catch